scripts/accumulator-server.py

```diff
@@ -81,42 +82,57 @@ def usage():
     Print usage message
     """
 
-    print 'Usage: %s --host <host> --port <port> --url <server url> --pretty-print -v -u' % os.path.basename(__file__)
-    print ''
-    print 'Parameters:'
-    print "  --host <host>: host to use database to use (default is '0.0.0.0')"
-    print "  --port <port>: port to use (default is 1028)"
-    print "  --mqttHost <host>: mqtt broker host to use (if not defined MQTT listening is not enabled)"
-    print "  --mqttPort <port>: mqtt broker port to use (default is 1883)"
-    print "  --mqttTopic <topic>: mqtt topic to use (default is accumulate_mqtt)"
-    print "  --url <server url>: server URL to use (default is /accumulate)"
-    print "  --pretty-print: pretty print mode"
-    print "  --https: start in https"
-    print "  --key: key file (only used if https is enabled)"
-    print "  --cert: cert file (only used if https is enabled)"
-    print "  -v: verbose mode"
-    print "  -u: print this usage message"
+    print(f"Usage: {os.path.basename(__file__)} --host <host> --port <port> --url <server url> --pretty-print -v -u")
+    print("""\nParameters:
+          --host <host>: host to use database to use (default is '0.0.0.0')"
+          --port <port>: port to use (default is 1028)"
+          --mqttHost <host>: mqtt broker host to use (if not defined MQTT listening is not enabled)"
+          --mqttPort <port>: mqtt broker port to use (default is 1883)"
+          --mqttTopic <topic>: mqtt topic to use (default is accumulate_mqtt)"
+          --url <server url>: server URL to use (default is /accumulate)"
+          --pretty-print: pretty print mode"
+          --https: start in https"
+          --key: key file (only used if https is enabled)"
+          --cert: cert file (only used if https is enabled)"
+          -v: verbose mode"
+          -u: print this usage message
+          """
+          )
 
 
 # This function is registered to be called upon termination
 def all_done():
     os.unlink(pidfile)
 
+
 # Default arguments
-port       = 1028
-host       = '0.0.0.0'
-mqtt_host  = None
-mqtt_port  = 1883
+port = 1028
+host = '0.0.0.0'
+mqtt_host = None
+mqtt_port = 1883
 mqtt_topic = 'accumulate_mqtt'
 server_url = '/accumulate'
-verbose    = 0
-pretty     = False
-https      = False
-key_file   = None
-cert_file  = None
+verbose = 0
+pretty = False
+https = False
+key_file = None
+cert_file = None
```

Comment:
I'd suggest to keep the same indentation in that part.

Reply:
I did this just to keep compliant with [PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)

But if you prefer to keep the indentation I'll add it again :wink:

Reply:
No, it's ok. Align with PEP8 is better.

NTC